### ISSUE TITLE
[IntelliJ] HDInsight clusters are false recognized as Reader role cluster if we sign in with Azure CLI

### ISSUE THREAD

**Issue by t-rufang** [microsoft/azure-tools-for-java#4474]
Repro steps:
1. Sign in with Device Code.
2. Refresh HDInsight node, we will get a list of clusters as below.
![image](https://user-images.githubusercontent.com/32627233/85947091-48b29300-b97b-11ea-8d8c-9cb48aa16415.png)

3. Sign out and re-signin with Azure CLI.
4. Refresh HDInsight node, we will get a list of clusters as below. You can find the clusters are recognized as Reader role cluster.

**Comment by t-rufang** [comment]
As you can see, we use the following code to generate a probe URL. And then we send a POST request with the URL to get cluster configuration. 
```java
    public Observable<Map> getClusterCoreSiteRequest(@NotNull final String clusterId) throws IOException {
        String managementURI = getAzureManager().getManagementURI();
        String url = URI.create(managementURI)
                .resolve(clusterId.replaceAll("/+$", "") + "/configurations/core-site").toString();
        return getHttp()
                .withUuidUserAgent()
                .get(url, null, null, Map.class);
    }
```
The problem is, if we sign in with Device code, the `managementURI` we get is `management.azure.com` and this is what we want. But if we sign in with Azure CLI, the `managementURI` we get would be `management.core.windows.net`. This causes the POST request returns `403` which further makes the cluster false recognized as Reader role cluster.

**Comment by jingyanjingyan** [comment]
Fix in build azure-toolkit-for-intellij-2019.3.develop.12471931.06-29-2020.zip, close bug.
